src/controllers/login.controller.ts

```diff
@@ -7,6 +8,8 @@ import { JwtResult } from '../typings/jwt.typings';
 export class LoginController {
   constructor(private loginService: LoginService) {}
 
+  @ApiTags('Login')
```

Comment:
I suggest adding in top of controller:
```suggestion
@ApiTags('Login')
export class LoginController {
  constructor(private loginService: LoginService) {}
```